Base/usr/share/man/man1/host.md

```diff
@@ -0,0 +1,26 @@
+## Name
+
+host - DNS lookup utility
+
+## Synopsis
+
+```**sh
+# host <name>
+```
+
+## Description
+
+`host` is a simple utility for performing DNS lookups. It is normally
+used to convert names to IP addresses and vice versa. When no arguments
+are given, `host` prints a short summary of its command line arguments.
+
+`name` is the domain name that is to be looked up. It can also be a
+dotted-decimal IPv4 address, in which case host will by default perform
```

Comment:
Please decide whether `host` (the name of the utility) should be monspace or not, and use one style throughout the page. Or better yet, look at how existing pages do it.

Reply:
"by default" - is there anything it would do with some options, or?..